modules/router-certificates.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="router-certificates_{context}"]
+= Router certificates
+
+[discrete]
+== Purpose
+
+The Ingress Operator uses certificates for:
+
+* Securing access to metrics for Prometheus.
+* Securing access to Routes.
```

Comment:
Route is capitalized here but not later on. Honestly I'm not sure whether it's supposed to be or not. And the guidance in our glossary isn't clear :) Says "Route" but the text is lower

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#route